LICENSE.md

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2018-2020 Michalis Zampetakis
+Copyright (c) 2015-2020 Thanasis Zografos
+Copyright (c) 2015-2020 Antonios Chariton
+Copyright (c) 2015-2020 Nikos Kampitakis
+Copyright (c) 2019-2020 Giannis Koutsaftakis
+Copyright (c) 2016-2020 Yorgos Saslis
```

Comment:
Please amend this line to be 2015-2019 

I believe sep 2019 was the date of requetsed time off

Reply:
Unless I am wrong and this is not required or even not appropriate as a change.  Might be wrong so I am open to corrections here.